src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -171,9 +171,17 @@ private static void StartProcessWithUTF8IOEncoding(Process process)
         {
             if (hasConsole)
             {
-                // Restore the original encodings
-                Console.InputEncoding = originalInputEncoding;
-                Console.OutputEncoding = originalOutputEncoding;
+                try
+                {
+                    // Restore the original encodings
+                    Console.InputEncoding = originalInputEncoding;
+                    Console.OutputEncoding = originalOutputEncoding;
+                }
+                catch (System.Exception)
```

Comment:
```suggestion
                catch (System.IO.IOException)
```
I would be more strict on the exception that is being caught to not accidentally hide additional problems.

Reply:
sg